packages/autobahn-xbr/lib/util.js

```diff
@@ -48,18 +48,25 @@ function pack_uint256 (value) {
 }
 
 
+function unpackUint256 (value) {
+    // FIXME: wont work in browser ?
```

Comment:
@oberstet I am having a hard time finding something that does the below stuff with `Uint8Array`, even though this code seems to work in the browser as a Vue app, I am not sure if it will always work (?)